### PR TITLE
docs: add changelog for 3.15.1

### PR DESCRIPTION
**Pull request by picodoth**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

1. Describe the source of requirement, like related issue link.

2. Describe the problem and the scenario.

### 💡 Solution

1. How to fix the problem, and list final API implementation and usage sample if that is an new feature.

2. GIF or snapshot should be provided if includes UI/interactive modification.

### 📝 Changelog description

> Describe changes from userside, and list all potential break changes or other risks.

1. English description

2. Chinese description (optional)

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+`2019-03-17`
+
+- 🌟 添加 less 变量 `@pagination-item-bg-active`。 [#14739](https://github.com/ant-design/ant-design/issues/14739)
+- 🌟 添加 less 变量 `@icon-color`。[#15343](https://github.com/ant-design/ant-design/issues/15343)
+- 🌟 添加两个 less 变量 `@btn-border-width` 和 `@btn-border-style` 来独立控制 Button 的样式。[#15397](https://github.com/ant-design/ant-design/issues/15343) [@searleb](https://github.com/searleb)
+- 🌟 添加 deps lint。[#15343](https://github.com/ant-design/ant-design/issues/15397)
+- 🌟 Modal 组件添加 `Modal.confirm({ icon: null })` 来隐藏默认的 Icon。[#15319](https://github.com/ant-design/ant-design/pull/15319)
+- List
+  - 🐞 保持 List 组件内容颜色与之前的版本一致。[#15296](https://github.com/ant-design/ant-design/issues/15296)
+  - 🐞 修复 List 的一些央视问题。[#15329](https://github.com/ant-design/ant-design/issues/15329)
```
Comment (imhele) [inline CHANGELOG.zh-CN.md]:
```diff
- 一些央视问题
+ 一些样式问题
```

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,36 @@ timeline: true
 
 ---
 
+## 3.15.1
+
+`2019-03-17`
+
+- 🌟 Add less variable `@pagination-item-bg-active`. [#14739](https://github.com/ant-design/ant-design/issues/14739)
+- 🌟 Add less variable `@icon-color`. [#15343](https://github.com/ant-design/ant-design/issues/15343)
+- 🌟 Add less variable `@btn-border-width` and `@btn-border-style` to control button style independently. [#15397](https://github.com/ant-design/ant-design/issues/15397) [@searleb](https://github.com/searleb)
```
Comment (chenshuai2144) [inline CHANGELOG.en-US.md]: 三个写一起？

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+- Badge
+  - 🐞 修复 Badge 组件应当支持自定义颜色。[#15349](https://github.com/ant-design/ant-design/issues/15349) [@DiamondYuan](https://github.com/DiamondYuan)
```
Comment (chenshuai2144) [inline CHANGELOG.zh-CN.md]: 修复 Badge 组件不支持自定义颜色的问题

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
+- 🐞 Correct suffix icon in Input.Password. [#15369](https://github.com/ant-design/ant-design/issues/15369) [melchior-voidwolf](https://github.com/melchior-voidwolf)
+- 🐞 Fix Layout has classname `ant-layout-has-sider` even if `hasFixer` is set to `false`. [#15396](https://github.com/ant-design/ant-design/pull/15396) [@SoraYama](https://github.com/SoraYama)
+- 🐞 Fix Divider alignment issue in PageHeader. [#15336](https://github.com/ant-design/ant-design/issues/15336)
+- 🐞 Fix Skeleton broken style. [#15413](https://github.com/ant-design/ant-design/issues/15413) [@Maktel](https://github.com/Maktel)
+- 🐞 Fixed a warning caused by setTimeout. [#15434](https://github.com/ant-design/ant-design/pull/15434) [@chiaweilee](https://github.com/chiaweilee)
+- Typescript
+  - 🌟 FormComponentProps added a generic type of form values. [#15355](https://github.com/ant-design/ant-design/pull/15355)
+  - 🌟 Export ConfigProviderProps interface. [#15438](https://github.com/ant-design/ant-design/issues/15438) [@DiamondYuan](https://github.com/DiamondYuan)
+  - 🐞 Add `onClick` prop for BreadcrumbItem. [#15331](https://github.com/ant-design/ant-design/pull/15331) [@tgxpuisb](https://github.com/tgxpuisb)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: BreadcrumbItem => Breadcrumb.Item

看其他都是 xx.xx 的形式

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
 ---
 
+## 3.15.1
+
+`2019-03-17`
+
+- 🌟 Add less variable `@pagination-item-bg-active`. [#14739](https://github.com/ant-design/ant-design/issues/14739)
+- 🌟 Add less variable `@icon-color`. [#15343](https://github.com/ant-design/ant-design/issues/15343)
+- 🌟 Add less variable `@btn-border-width` and `@btn-border-style` to control button style independently. [#15397](https://github.com/ant-design/ant-design/issues/15397) [@searleb](https://github.com/searleb)
+- 🌟 Add deps lint. [#15407](https://github.com/ant-design/ant-design/issues/15407)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: 这个改成 `修复了部分组件 tree shaking 失效的问题` 可能会更直观

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
+- 🌟 Modal support `Modal.confirm({ icon: null })` to hide default Icon. [#15319](https://github.com/ant-design/ant-design/pull/15319)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: support => supports

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
+- 🐞 Fix Divider alignment issue in PageHeader. [#15336](https://github.com/ant-design/ant-design/issues/15336)
+- 🐞 Fix Skeleton broken style. [#15413](https://github.com/ant-design/ant-design/issues/15413) [@Maktel](https://github.com/Maktel)
+- 🐞 Fixed a warning caused by setTimeout. [#15434](https://github.com/ant-design/ant-design/pull/15434) [@chiaweilee](https://github.com/chiaweilee)
+- Typescript
+  - 🌟 FormComponentProps added a generic type of form values. [#15355](https://github.com/ant-design/ant-design/pull/15355)
+  - 🌟 Export ConfigProviderProps interface. [#15438](https://github.com/ant-design/ant-design/issues/15438) [@DiamondYuan](https://github.com/DiamondYuan)
+  - 🐞 Add `onClick` prop for BreadcrumbItem. [#15331](https://github.com/ant-design/ant-design/pull/15331) [@tgxpuisb](https://github.com/tgxpuisb)
+  - 🐞 Add `style` prop for Steps.Step component. [#15358](https://github.com/ant-design/ant-design/issues/15358) [@pavolgolias](https://github.com/pavolgolias)
+  - 🐞 Fixed `itemRender` definition of Pagination. [#15384](https://github.com/ant-design/ant-design/issues/15384) [@DiamondYuan](https://github.com/DiamondYuan)
```
Comment (zy410419243) [inline CHANGELOG.en-US.md]: Fixed => Fix

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+  - 🐞 修复 Badge 组件内带小圆点时的高度问题。[#15367](https://github.com/ant-design/ant-design/issues/15367)
+- 🐞 修复 Tabs 组件在卡片模式下 tabbar 的样式。[#15297](https://github.com/ant-design/ant-design/issues/15297) [@rinick](https://github.com/rinick)
+- 🐞 修复 Calendar 无法切换类型的问题。[#15334](https://github.com/ant-design/ant-design/issues/15334) [@zy410419243](https://github.com/zy410419243)
+- 🐞 修复 Avatar 组件设置了 `display: none` 的时候无法计算偏移量的问题。[#15351](https://github.com/ant-design/ant-design/pull/15351) [@ppbl](https://github.com/ppbl)
+- 🐞 修复 Input.Password 组件的后置 Icon。[#15369](https://github.com/ant-design/ant-design/issues/15369) [melchior-voidwolf](https://github.com/melchior-voidwolf)
+- 🐞 修复即使在指定 Layout 的 `hasSider` 属性为 `false` 的时候依然会加 `ant-layout-has-sider` 的 classname 的问题[#15396](https://github.com/ant-design/ant-design/pull/15396) [@SoraYama](https://github.com/SoraYama)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: 修复即使在... => 修复 Layout 即使在...

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,37 @@ timeline: true
...
+- 🐞 修复 Divider 在 PageHeader 组件中的对齐问题。[#15336](https://github.com/ant-design/ant-design/issues/15336)
+- 🐞 修复 Menu 收缩后，SubMenu 异常渲染的问题。[#15409](https://github.com/ant-design/ant-design/pull/15409) [@zy410419243](https://github.com/zy410419243)
+- 🐞 修复 Skeleton 组件的样式问题。[#15413](https://github.com/ant-design/ant-design/issues/15413) [@Maktel](https://github.com/Maktel)
+- 🐞 修复一个 setTimeout 引起的 warning。[#15434](https://github.com/ant-design/ant-design/pull/15434) [@chiaweilee](https://github.com/chiaweilee)
```
Comment (zy410419243) [inline CHANGELOG.zh-CN.md]: 一个...

File: CHANGELOG.en-US.md [hunk 10]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
+- List
+  - 🐞 Keep same content color of List like previous version. [#15296](https://github.com/ant-design/ant-design/issues/15296)
+  - 🐞 Fix some List Item styling issues. [#15329](https://github.com/ant-design/ant-design/issues/15329)
+- Badge
+  - 🐞 Fix Badge should support color style. [#15349](https://github.com/ant-design/ant-design/issues/15349) [@DiamondYuan](https://github.com/DiamondYuan)
+  - 🐞 Fix height problem of Badge with dot. [#15367](https://github.com/ant-design/ant-design/issues/15367)
+- 🐞 Fix tabbar style of Tabs component in card mode. [#15297](https://github.com/ant-design/ant-design/issues/15297) [@rinick](https://github.com/rinick)
+- 🐞 Fix Calendar can't switch type. [#15334](https://github.com/ant-design/ant-design/issues/15334) [@zy410419243](https://github.com/zy410419243)
+- 🐞 Fix Avatar component cannot calculate the offset when `display: none` is set. [#15351](https://github.com/ant-design/ant-design/pull/15351) [@ppbl](https://github.com/ppbl)
+- 🐞 Correct suffix icon in Input.Password. [#15369](https://github.com/ant-design/ant-design/issues/15369) [melchior-voidwolf](https://github.com/melchior-voidwolf)
```
Comment (ztplz) [inline CHANGELOG.en-US.md]: 少了个 `@`

**Review by ztplz — APPROVED** [review]

**Comment by zombieJ** [comment]
url 统一连到 PR 上，没有才连到 issue 上。

File: CHANGELOG.en-US.md (hunk 5 above)
Comment (zombieJ) [reply CHANGELOG.en-US.md]: 改成 “修复 PageHeader tree shaking 失效的问题” 好了，deps lint 对用户来说是无感知的。

File: CHANGELOG.en-US.md [hunk 11]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
+- 🐞 Fix Layout has classname `ant-layout-has-sider` even if `hasFixer` is set to `false`. [#15396](https://github.com/ant-design/ant-design/pull/15396) [@SoraYama](https://github.com/SoraYama)
+- 🐞 Fix Divider alignment issue in PageHeader. [#15336](https://github.com/ant-design/ant-design/issues/15336)
+- 🐞 Fix Skeleton broken style. [#15413](https://github.com/ant-design/ant-design/issues/15413) [@Maktel](https://github.com/Maktel)
+- 🐞 Fixed a warning caused by setTimeout. [#15434](https://github.com/ant-design/ant-design/pull/15434) [@chiaweilee](https://github.com/chiaweilee)
```
[the remark this answers is not in the document]
Comment (zombieJ) [reply CHANGELOG.en-US.md]: 这个是 site fix，不用出现在 changelog 里。

File: CHANGELOG.en-US.md (hunk 4 above)
Comment (zombieJ) [reply CHANGELOG.en-US.md]: typescript 定义更新都放最下面。参考：https://github.com/ant-design/ant-design/releases/tag/3.14.0

File: CHANGELOG.en-US.md [hunk 12]
```diff
@@ -15,6 +15,36 @@ timeline: true
...
+- Typescript
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: 调整了多处 TypeScript 的类型
Adjusted multiple TypeScript types
ref:https://github.com/ant-design/ant-design/releases/tag/3.14.0

**Comment by picodoth** [comment]
> url 统一连到 PR 上，没有才连到 issue 上。

@zombieJ 要不要先搁置争议啊

**Comment by picodoth** [comment]
issue link 都改成 pr link 了，CI 之前绿的，就合了。npm run pub 也会确保 ci 都 ok。

File: CHANGELOG.zh-CN.md [hunk 13]
```diff
@@ -15,6 +15,34 @@ timeline: true
 
 ---
 
+## 3.15.1
+
+`2019-03-17`
+
+- 🌟 添加了四个 less 变量 `@pagination-item-bg-active`、`@icon-color`、`@btn-border-width` 以及 `@btn-border-style`。 [#15302](https://github.com/ant-design/ant-design/pull/15302)、[#15345](https://github.com/ant-design/ant-design/pull/15345)、[#15397](https://github.com/ant-design/ant-design/pull/15397) [@searleb](https://github.com/searleb)
+- 🌟 修复 PageHeader tree shaking 失效的问题。[#15354](https://github.com/ant-design/ant-design/pull/15354)
+- 🌟 Modal 组件添加 `Modal.confirm({ icon: null })` 来隐藏默认的 Icon。[#15319](https://github.com/ant-design/ant-design/pull/15319)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 这个是提交到 feature 上的，不在 master 上。

File: CHANGELOG.zh-CN.md [hunk 14]
```diff
@@ -15,6 +15,34 @@ timeline: true
 
 ---
 
+## 3.15.1
+
+`2019-03-17`
+
+- 🌟 添加了四个 less 变量 `@pagination-item-bg-active`、`@icon-color`、`@btn-border-width` 以及 `@btn-border-style`。 [#15302](https://github.com/ant-design/ant-design/pull/15302)、[#15345](https://github.com/ant-design/ant-design/pull/15345)、[#15397](https://github.com/ant-design/ant-design/pull/15397) [@searleb](https://github.com/searleb)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: https://github.com/ant-design/ant-design/pull/15397 也是提交到 feature 的。

File: CHANGELOG.en-US.md [hunk 15]
```diff
@@ -15,6 +15,33 @@ timeline: true
 
 ---
 
+## 3.15.1
+
+`2019-03-17`
+
+- 🌟 Add four less variables `@pagination-item-bg-active`, `@icon-color`, `@btn-border-width` and `@btn-border-style`. [#15302](https://github.com/ant-design/ant-design/pull/15302), [#15343](https://github.com/ant-design/ant-design/issues/15343), [#15397](https://github.com/ant-design/ant-design/issues/15397) [@searleb](https://github.com/searleb)
+- 🌟 Fix tree shaking not working for PageHeader. [#15354](https://github.com/ant-design/ant-design/pull/15354)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: Fix => 🐞

File: CHANGELOG.zh-CN.md [hunk 16]
```diff
@@ -15,6 +15,34 @@ timeline: true
...
+- 🐞 修复 Calendar 无法切换类型的问题。[#15338](https://github.com/ant-design/ant-design/pull/15338) [@zy410419243](https://github.com/zy410419243)
+- 🐞 修复 Avatar 组件设置了 `display: none` 的时候无法计算偏移量的问题。[#15351](https://github.com/ant-design/ant-design/pull/15351) [@ppbl](https://github.com/ppbl)
+- 🐞 修复 Input.Password 组件的后置 Icon。[#15381](https://github.com/ant-design/ant-design/pull/15381) [@melchior-voidwolf](https://github.com/melchior-voidwolf)
+- 🐞 修复 Layout 即使在 `hasSider` 属性为 `false` 的时候依然会加 `ant-layout-has-sider` 的 classname 的问题[#15396](https://github.com/ant-design/ant-design/pull/15396) [@SoraYama](https://github.com/SoraYama)
+- 🐞 修复 Divider 在 PageHeader 组件中的对齐问题。[#15400](https://github.com/ant-design/ant-design/pull/15400)
+- 🐞 修复 Menu 收缩后，SubMenu 异常渲染的问题。[#15409](https://github.com/ant-design/ant-design/pull/15409) [@zy410419243](https://github.com/zy410419243)
+- 🐞 修复 Skeleton 组件的样式问题。[#15421](https://github.com/ant-design/ant-design/pull/15421) [@Maktel](https://github.com/Maktel)
+- 调整了多处 Typescript 类型
+  - 🌟 FormComponentProps 新增 Form 值的范型。[#15355](https://github.com/ant-design/ant-design/pull/15355)
```
Comment (imhele) [inline CHANGELOG.zh-CN.md]:
```diff
- 新增 Form 值的范型
+ 新增 Form 值的泛型
```

**Pull request merged**